finalPrep/README.md

```diff
@@ -32,23 +32,29 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+- ![Screen Shot 2021-12-17 at 6 05 59 PM](https://user-images.githubusercontent.com/3982238/146623794-2558406b-12ea-478d-a5ec-e7ac0cc5d047.png)
```

Comment:
I'd love to see you get much more specific here. Including:
- When are you going to eat? Do anything for yourself (no screen time, outdoors, dedicated phone call catch up time with friends/fam, etc - whatever you need as a human)?
- You know you need to review and preview projects; that goes best when students have a routine and dedicated time. What about a 20-30 minute block for each of those before/after each day of classes?
- What's happening on Sundays? Is that off-limit-to-Turing?